airbyte-webapp/src/components/ui/PillSelect/PillButton.tsx

```diff
@@ -5,20 +5,33 @@ import classNames from "classnames";
 import { Text } from "../Text";
 import styles from "./PillButton.module.scss";
 
+export type PillButtonVariant = "grey" | "blue" | "green" | "red" | "strong-red";
+
+const STYLES_BY_VARIANT: Readonly<Record<PillButtonVariant, string>> = {
+  grey: styles.grey,
+  blue: styles.blue,
+  green: styles.green,
+  red: styles.red,
+  "strong-red": styles.strongRed,
```

Comment:
I'm curious what the need for two red variants is? It seems to me like it could be potentially confusing UX

Reply:
Discussed over slack. We don't have a good name for these in the design file, and there are two red and blue variants with similar patterns.